src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.targets

```diff
@@ -211,7 +211,11 @@ Copyright (c) .NET Foundation. All rights reserved.
     <GenerateSatelliteAssemblies EmbedResources="@(_SatelliteAssemblyResourceInputs)"
                                  AssemblyInfoFile="$(_AssemblyInfoFile)"
                                  OutputAssembly="$(_OutputAssembly)"
-                                 References="@(Reference)">
+                                 References="@(ReferencePath)"
```

Comment:
Should this target have `DependsOnTargets="ResolveAssemblyReference"` since it uses the output of RAR now?  Also, do the ReferencePaths need to be specified as an input to the target?